roles/apache/defaults/main.yml

```
@@ -0,0 +1,4 @@
+---
+# defaults file for apache
+
+backup_dir: /var/apache_backup
```

The README file is very useful, but I think that if we describe here all the variables with indicative values will be completely clear.

```suggestion
backup_dir: /var/apache_backup

httpd_document_root: /var/www/html/

httpd_rev_proxy:
    ip: 127.0.0.1
    port: 8080

httpd_extra_conf: |
    <Directory "/var/www/html">
      Options Indexes FollowSymLinks
      AllowOverride None
      Require all granted
    </Directory>
```

I also suggest using `document_root` instead of `static_dir`.

Finally, I suggest splitting this dictionary into **independent variables** so that they are easy to **overlap**.

---

I didn't add all the variables to default.yml because it will produce incorrect configuration files when the playbook runs. For example, here it will create a virtualhost conf file which will serve a static directory, it will serve a reverse proxy and it will have a Directory tag

---

Forgive me for the confusion, I mean to be in comments, but have some sample values.

_I corrected the above suggestion with all the variables in the comments._